### PR TITLE
issue-615: Add test case for SofaServiceBeanPostProcessor.

### PR DESCRIPTION
**Pull request by HoldDie**
Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    @Test
+    public void testPostProcessAfterInitialization() throws Exception {
+        ServiceFactoryBean serviceFactoryBean = new ServiceFactoryBean();
+        Class<?> c = serviceFactoryBean.getClass();
+        Field field = c.getDeclaredField("service");
+        field.setAccessible(true);
+        ServiceImpl service = new ServiceImpl("uniqueId", SofaService.class, new SoulSofaServiceImpl());
+        field.set(serviceFactoryBean, service);
+
+        Field field1 = c.getSuperclass().getDeclaredField("interfaceClass");
```
Comment (tydhot) [inline soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java:99]: don't use that way to name a variable

File: soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+import static io.undertow.Handlers.path;
+
+/**
+ * Test case for SofaServiceBeanPostProcessor.
+ *
+ * @author HoldDie
+ */
+@RunWith(MockitoJUnitRunner.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class SofaServiceBeanPostProcessorTest {
```
Comment (tydhot) [inline soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java:51]: add final

**Review by tydhot — CHANGES_REQUESTED** [review]

File: soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java (hunk 2 above)
Comment (HoldDie) [reply soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java:51]: got it.

File: soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java (hunk 1 above)
Comment (HoldDie) [reply soul-client/soul-client-sofa/src/test/java/org/dromara/soul/client/sofa/SofaServiceBeanPostProcessorTest.java]: ok.

**Review by tydhot — APPROVED** [review]

**Pull request merged**